docs/gallery.rst

```diff
@@ -0,0 +1,78 @@
+.. -*- mode: rst -*-
+
+====================
+Example Gallery
+====================
+
+Yellowbrick Examples
+
+Features Analysis
+-----------------
+
+.. image:: api/features/images/radviz.png
+    :width: 200px
+    :height: 100px
+    :alt: Features Analysis adViz Visualizer
+    :target: api/features/radviz.html#radviz-visualizer
+
+.. image:: api/features/images/rank1d_shapiro.png
+    :width: 200px
+    :height: 100px
```

Comment:
Wow, this is all it takes to get the grid layout! I was worried it would be much more difficult; thanks for taking a look at this!